bin/millau/node/src/chain_spec.rs

```diff
@@ -134,54 +133,38 @@ impl Alternative {
 /// purposes), are all available on these chains.
 fn endowed_accounts() -> Vec<AccountId> {
 	vec![
```

Comment:
Nit: Can we define these names in an enum (or consts) in order to be able to reuse them ?